docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.8"
+services:
+  server:
+    build: .
+    image: ottomated/crewlink-server:build
+    container_name: crewlinkserver
+    ports:
+        - 9736:9736
```

Comment:
The compose spec specifies that `HOST:CONTAINER` port mappings should be quoted to avoid conflicts: https://github.com/compose-spec/compose-spec/blob/master/spec.md#short-syntax-2

Reply:
Looks like I'm learnings things with this MR :D